### ISSUE TITLE
Bug: Gremlins won't execute in Cypress test

### ISSUE THREAD

**Issue by Flaxline** [marmelab/gremlins.js#165]
-   `gremlins.js` version: 2.1.1
-   `node` version: 12.6.2
-   `npm` (or `yarn`) version: 6.14.4
-   browser version: Chrome 83

- cypress version: 4.4.1

## Steps To Reproduce

1. Angular Web Application, working as expected
2. e2e-Test with Cypress against this Webapp
3. Start Gremlins in Constructor of Webapp - works as expected
4. Start Gremlins in a Cypress Testcase instead (just like in Webapp itself - import in the spec.ts and call gremlins.createHorde().unleash())

## The current behavior

There's an error: `Uncaught (in promise) TypeError: Cannot read property 'dispatchEvent' of null` when I use createHorde without further configuration.
When I configure to just use Clicker/Toucher, the error doesn't occur, but there are also no visible Actions from these Gremlins (not in the app I'm testing, nor the Cypress UI).
As I said, FPS Mogwai is executed in both cases and prints its outputs to console as expected.

There's no further error messages accessible.

I tried printing gremlins-object and horde-object to console, both seemed to be set correctly, containing the functions I called. 

## The expected behavior

Expected behaviour would be the Gremlins executing at least SOMEWHERE, or an error message showing the problem.

In some older issue, the use of Cypress was mentioned, but not in any detail: https://github.com/marmelab/gremlins.js/issues/159
Since there's an issue stating it's not possible to run gremlins in an iframe (https://github.com/marmelab/gremlins.js/issues/2), but Cypress opens the app to test in an iFrame, I am not sure if that's the reason of my problem already.

### Suggested solution:

I'm mostly looking for confirmation that the usage of iFrame is the issue here, if that's the case. But as mentioned above, someone already seemed to use gremlins within cypress, so I'm not that sure of it anymore.

**Comment by HALLERPierre** [comment]
Hi and thank you for your issue :smile: !

I was able to reproduce. The problem is that gremlins.js use the current window object, which is the cypress' window. To run gremlins.js inside a cypress test, we need to allow a custom window to be passed to gremlins.js.

This is not currently possible, but I opened a PR that will allow that: https://github.com/marmelab/gremlins.js/pull/166
It should be available in the next release.

Then you'll be able to create a horde inside a cypress test as described here : https://github.com/marmelab/gremlins.js/blob/allow-custom-window/README.md#cypress
